shared/src/utils/validators/index.ts

```diff
@@ -0,0 +1,6 @@
+// TODO: Propagate the errors from subobjects
```

Comment:
This is still a problem?

Reply:
Yes, for example the `isReachableTransferPoints` validator would only return `false` without specifying what went wrong.